screens/Treatments.js

```diff
@@ -1,24 +1,46 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { StyleSheet, View, FlatList, Button, Text, Modal, TouchableOpacity } from 'react-native';
 import { grey } from 'color-name';
```

Comment:
GlobalColors import?

```suggestion
import { GlobalColors } from '../assets/GlobalStyle';
```